app/controllers/searches_controller.rb

```diff
@@ -5,18 +5,55 @@ class SearchesController < CatalogController
 
   def index
     @search = FileSearch.new(params, catalog_query: self)
-    @results = @search.results
+    get_audios
+    get_videos
+    get_images
+    get_articles
     # page = params[:page] || 1
     # per_page = params[:per_page] || 10
     # resource_type = params[:t].singularize.capitalize
     #
   end
 
+  def videos
+    get_videos
+    render layout: false
+  end
+
+  def images
+    get_images
+    render layout: false
+  end
+
+  def articles
+    get_articles
+    render layout: false
+  end
+
+  def audios
+    get_audios
+    render layout: false
+  end
+
   private
 
-  # def do_search(query, filters, resource_type, page, per_page)
-  #   facets = {desc_metadata__resource_type_sim: resource_type}
-  #   facets.merge!(filters_to_query_values(filters)) unless !filters || filters.empty?
-  #   q_params = {q: query, f: facets, page: page, per_page: per_page }
-  # end
+  def get_audios
+    @audios_search = FileSearch.new(params.merge(:resource_types => ['Audio']), catalog_query: self)
+    @audios = @audios_search.files
+  end
+
+  def get_videos
+    @videos_search = FileSearch.new(params.merge(:resource_types => ['Video']), catalog_query: self)
+    @videos = @videos_search.files
+  end
+
+  def get_articles
+    @articles_search = FileSearch.new(params.merge(:resource_types => ['Article']), catalog_query: self)
+    @articles = @articles_search.files
+  end
+
+  def get_images
+    @images_search = FileSearch.new(params.merge(:resource_types => ['Image']), catalog_query: self)
+    @images = @images_search.files
+  end
```

Comment:
Not at the moment. This is probably an interim step leading to separating the resources in a more meaningful way. Currently each of these methods only gets called once per request. It's mostly just for deduplication and likely to go away in the near future.